clients/src/main/java/org/apache/kafka/clients/NetworkClient.java

```diff
@@ -855,10 +855,11 @@ private void handleTimedOutConnections(List<ClientResponse> responses, long now)
         List<String> nodes = connectionStates.nodesWithConnectionSetupTimeout(now);
         for (String nodeId : nodes) {
             this.selector.close(nodeId);
-            log.info(
-                "Disconnecting from node {} due to socket connection setup timeout. " +
+            log.warn(
```

Comment:
Why is this a warning? It is expected to happen in a number of expected situations (like cluster rolls). The additional information seems useful.

Reply:
got it, i'll revert WARN to INFO.